[2024-09-10 22:46:31,093:INFO: main: Welcome to our custom log]
[2024-09-13 18:36:57,869:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-13 18:36:57,873:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-13 18:36:57,874:INFO: common: created directory at: artifacts]
[2024-09-13 18:36:57,875:INFO: common: created directory at: artifacts/data_ingestion]
[2024-09-13 18:37:37,123:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-13 18:37:37,124:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-13 18:37:37,125:INFO: common: created directory at: artifacts]
[2024-09-13 18:37:37,125:INFO: common: created directory at: artifacts/data_ingestion]
[2024-09-13 18:37:43,495:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-13 18:37:43,497:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-13 18:37:43,497:INFO: common: created directory at: artifacts]
[2024-09-13 18:37:43,497:INFO: common: created directory at: artifacts/data_ingestion]
[2024-09-13 18:40:38,466:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-13 18:40:38,468:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-13 18:40:38,468:INFO: common: created directory at: artifacts]
[2024-09-13 18:40:38,469:INFO: common: created directory at: artifacts/data_ingestion]
[2024-09-13 18:40:38,469:INFO: 2485677886: Downloading data from https://drive.google.com/file/d/1YKuwii_lywWI_03kOtWC8HF7xy1w2-93/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-09-13 18:41:57,923:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-13 18:41:57,925:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-13 18:41:57,925:INFO: common: created directory at: artifacts]
[2024-09-13 18:41:57,926:INFO: common: created directory at: artifacts/data_ingestion]
[2024-09-13 18:41:57,927:INFO: 606855465: Downloading data from https://drive.google.com/file/d/1YKuwii_lywWI_03kOtWC8HF7xy1w2-93/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-09-13 18:42:09,812:INFO: 606855465: Downloaded data from https://drive.google.com/file/d/1YKuwii_lywWI_03kOtWC8HF7xy1w2-93/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-09-13 19:08:14,150:INFO: main: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2024-09-13 19:08:14,151:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-13 19:08:14,151:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-13 19:08:14,152:INFO: common: created directory at: artifacts]
[2024-09-13 19:08:14,152:INFO: common: created directory at: artifacts/data_ingestion]
[2024-09-13 19:08:14,152:INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1YKuwii_lywWI_03kOtWC8HF7xy1w2-93/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-09-13 19:08:26,654:INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1YKuwii_lywWI_03kOtWC8HF7xy1w2-93/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-09-13 19:08:27,043:INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<<

x==========x]
[2024-09-14 17:26:53,562:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-14 17:26:53,565:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-14 17:26:53,566:INFO: common: created directory at: artifacts]
[2024-09-14 17:26:53,567:INFO: common: created directory at: artifacts/prepare_base_model]
[2024-09-14 17:27:16,578:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-14 17:27:16,580:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-14 17:27:16,581:INFO: common: created directory at: artifacts]
[2024-09-14 17:27:16,581:INFO: common: created directory at: artifacts/prepare_base_model]
[2024-09-14 17:27:55,309:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-14 17:27:55,310:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-14 17:27:55,311:INFO: common: created directory at: artifacts]
[2024-09-14 17:27:55,312:INFO: common: created directory at: artifacts/prepare_base_model]
[2024-09-14 17:29:43,880:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-14 17:29:43,882:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-14 17:29:43,883:INFO: common: created directory at: artifacts]
[2024-09-14 17:29:43,883:INFO: common: created directory at: artifacts/prepare_base_model]
[2024-09-14 17:30:05,004:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-14 17:30:05,006:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-14 17:30:05,006:INFO: common: created directory at: artifacts]
[2024-09-14 17:30:05,006:INFO: common: created directory at: artifacts/prepare_base_model]
[2024-09-14 17:34:29,956:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-14 17:34:29,959:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-14 17:34:29,959:INFO: common: created directory at: artifacts]
[2024-09-14 17:34:29,960:INFO: common: created directory at: artifacts/prepare_base_model]
[2024-09-14 17:36:43,896:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-14 17:36:43,898:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-14 17:36:43,899:INFO: common: created directory at: artifacts]
[2024-09-14 17:36:43,899:INFO: common: created directory at: artifacts/prepare_base_model]
[2024-09-14 17:36:54,801:WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-14 17:36:54,837:WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-14 17:53:47,563:INFO: main: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2024-09-14 17:53:47,564:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-14 17:53:47,564:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-14 17:53:47,564:INFO: common: created directory at: artifacts]
[2024-09-14 17:53:47,564:INFO: common: created directory at: artifacts/data_ingestion]
[2024-09-14 17:53:47,564:INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1YKuwii_lywWI_03kOtWC8HF7xy1w2-93/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-09-14 17:54:11,984:INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1YKuwii_lywWI_03kOtWC8HF7xy1w2-93/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-09-14 17:54:12,382:INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<<

x==========x]
[2024-09-14 17:54:12,382:INFO: main: ********************]
[2024-09-14 17:54:12,382:INFO: main: >>>>>>> stage Prepare Base model started <<<<<<<]
[2024-09-14 17:54:12,383:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-14 17:54:12,383:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-14 17:54:12,383:INFO: common: created directory at: artifacts]
[2024-09-14 17:54:12,383:INFO: common: created directory at: artifacts/prepare_base_model]
[2024-09-14 17:54:12,537:WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-14 17:54:12,571:WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-14 17:54:12,597:INFO: main: >>>>>>> stage Prepare Base model completed <<<<<<<

x==========x]
[2024-09-14 18:38:40,257:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-14 18:38:40,260:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-14 18:38:40,261:INFO: common: created directory at: artifacts]
[2024-09-14 18:38:40,262:INFO: common: created directory at: artifacts/training]
[2024-09-14 18:38:40,521:WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-09-14 18:39:07,096:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-14 18:39:07,098:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-14 18:39:07,098:INFO: common: created directory at: artifacts]
[2024-09-14 18:39:07,099:INFO: common: created directory at: artifacts/training]
[2024-09-14 18:39:07,161:WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-09-14 18:41:42,038:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-14 18:41:42,040:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-14 18:41:42,040:INFO: common: created directory at: artifacts]
[2024-09-14 18:41:42,041:INFO: common: created directory at: artifacts/training]
[2024-09-14 18:41:42,170:WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-09-14 18:44:07,299:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-14 18:44:07,302:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-14 18:44:07,303:INFO: common: created directory at: artifacts]
[2024-09-14 18:44:07,303:INFO: common: created directory at: artifacts/training]
[2024-09-14 18:44:07,426:WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-09-14 18:49:20,550:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-14 18:49:20,552:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-14 18:49:20,552:INFO: common: created directory at: artifacts]
[2024-09-14 18:49:20,553:INFO: common: created directory at: artifacts/training]
[2024-09-14 18:49:20,633:WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-09-14 18:49:55,452:WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-14 18:50:31,187:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-14 18:50:31,189:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-14 18:50:31,190:INFO: common: created directory at: artifacts]
[2024-09-14 18:50:31,190:INFO: common: created directory at: artifacts/training]
[2024-09-14 18:50:31,256:WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-09-14 18:53:21,393:WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-14 19:08:53,020:INFO: main: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2024-09-14 19:08:53,020:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-14 19:08:53,021:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-14 19:08:53,021:INFO: common: created directory at: artifacts]
[2024-09-14 19:08:53,021:INFO: common: created directory at: artifacts/data_ingestion]
[2024-09-14 19:08:53,021:INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1YKuwii_lywWI_03kOtWC8HF7xy1w2-93/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-09-14 19:09:08,333:INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1YKuwii_lywWI_03kOtWC8HF7xy1w2-93/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-09-14 19:09:08,698:INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<<

x==========x]
[2024-09-14 19:09:08,698:INFO: main: ********************]
[2024-09-14 19:09:08,698:INFO: main: >>>>>>> stage Prepare Base model started <<<<<<<]
[2024-09-14 19:09:08,699:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-14 19:09:08,700:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-14 19:09:08,700:INFO: common: created directory at: artifacts]
[2024-09-14 19:09:08,700:INFO: common: created directory at: artifacts/prepare_base_model]
[2024-09-14 19:09:08,862:WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-14 19:09:08,894:WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-14 19:09:08,911:INFO: main: >>>>>>> stage Prepare Base model completed <<<<<<<

x==========x]
[2024-09-14 19:09:08,911:INFO: main: ********************]
[2024-09-14 19:09:08,911:INFO: main: >>>>>>> stage Training started <<<<<<<]
[2024-09-14 19:09:08,912:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-14 19:09:08,912:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-14 19:09:08,912:INFO: common: created directory at: artifacts]
[2024-09-14 19:09:08,913:INFO: common: created directory at: artifacts/training]
[2024-09-14 19:09:08,965:WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-09-14 19:16:40,613:INFO: main: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2024-09-14 19:16:40,614:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-14 19:16:40,615:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-14 19:16:40,615:INFO: common: created directory at: artifacts]
[2024-09-14 19:16:40,615:INFO: common: created directory at: artifacts/data_ingestion]
[2024-09-14 19:16:40,615:INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1YKuwii_lywWI_03kOtWC8HF7xy1w2-93/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-09-14 19:16:57,998:INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1YKuwii_lywWI_03kOtWC8HF7xy1w2-93/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-09-14 19:16:58,368:INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<<

x==========x]
[2024-09-14 19:16:58,368:INFO: main: ********************]
[2024-09-14 19:16:58,369:INFO: main: >>>>>>> stage Prepare Base model started <<<<<<<]
[2024-09-14 19:16:58,369:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-14 19:16:58,369:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-14 19:16:58,370:INFO: common: created directory at: artifacts]
[2024-09-14 19:16:58,370:INFO: common: created directory at: artifacts/prepare_base_model]
[2024-09-14 19:16:58,524:WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-14 19:16:58,556:WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-14 19:16:58,573:INFO: main: >>>>>>> stage Prepare Base model completed <<<<<<<

x==========x]
[2024-09-14 19:16:58,574:INFO: main: ********************]
[2024-09-14 19:16:58,574:INFO: main: >>>>>>> stage Training started <<<<<<<]
[2024-09-14 19:16:58,574:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-14 19:16:58,575:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-14 19:16:58,575:INFO: common: created directory at: artifacts]
[2024-09-14 19:16:58,575:INFO: common: created directory at: artifacts/training]
[2024-09-14 19:16:58,616:WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-09-14 19:19:43,363:WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-14 19:19:43,394:INFO: main: >>>>>>> stage Training completed <<<<<<<

x==========x]
[2024-09-15 14:19:52,229:INFO: main: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2024-09-15 14:19:52,230:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-15 14:19:52,231:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-15 14:19:52,231:INFO: common: created directory at: artifacts]
[2024-09-15 14:19:52,231:INFO: common: created directory at: artifacts/data_ingestion]
[2024-09-15 14:19:52,231:INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1YKuwii_lywWI_03kOtWC8HF7xy1w2-93/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-09-15 14:20:11,976:INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1YKuwii_lywWI_03kOtWC8HF7xy1w2-93/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-09-15 14:20:12,385:INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<<

x==========x]
[2024-09-15 14:20:12,385:INFO: main: ********************]
[2024-09-15 14:20:12,385:INFO: main: >>>>>>> stage Prepare Base model started <<<<<<<]
[2024-09-15 14:20:12,386:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-15 14:20:12,387:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-15 14:20:12,387:INFO: common: created directory at: artifacts]
[2024-09-15 14:20:12,387:INFO: common: created directory at: artifacts/prepare_base_model]
[2024-09-15 14:20:12,544:WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-15 14:20:12,574:WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-15 14:20:12,593:INFO: main: >>>>>>> stage Prepare Base model completed <<<<<<<

x==========x]
[2024-09-15 14:20:12,593:INFO: main: ********************]
[2024-09-15 14:20:12,593:INFO: main: >>>>>>> stage Training started <<<<<<<]
[2024-09-15 14:20:12,594:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-15 14:20:12,594:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-15 14:20:12,594:INFO: common: created directory at: artifacts]
[2024-09-15 14:20:12,594:INFO: common: created directory at: artifacts/training]
[2024-09-15 14:20:12,646:WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-09-15 14:22:59,074:WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-15 14:22:59,102:INFO: main: >>>>>>> stage Training completed <<<<<<<

x==========x]
[2024-09-15 14:22:59,102:INFO: main: ********************]
[2024-09-15 14:22:59,102:INFO: main: >>>>>>> stage Evaluation stage started <<<<<<<]
[2024-09-15 14:22:59,103:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-15 14:22:59,104:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-15 14:22:59,104:INFO: common: created directory at: artifacts]
[2024-09-15 14:22:59,148:WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-09-15 14:23:08,911:INFO: common: json file saved at: scores.json]
[2024-09-15 14:23:08,911:INFO: common: json file saved at: scores.json]
[2024-09-15 14:23:12,357:INFO: main: >>>>>>> stage Evaluation stage completed <<<<<<<

x==========x]
[2024-09-15 14:26:10,092:INFO: main: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2024-09-15 14:26:10,093:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-15 14:26:10,093:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-15 14:26:10,093:INFO: common: created directory at: artifacts]
[2024-09-15 14:26:10,093:INFO: common: created directory at: artifacts/data_ingestion]
[2024-09-15 14:26:10,094:INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1YKuwii_lywWI_03kOtWC8HF7xy1w2-93/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-09-15 14:26:29,270:INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1YKuwii_lywWI_03kOtWC8HF7xy1w2-93/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-09-15 14:26:29,652:INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<<

x==========x]
[2024-09-15 14:26:29,652:INFO: main: ********************]
[2024-09-15 14:26:29,652:INFO: main: >>>>>>> stage Prepare Base model started <<<<<<<]
[2024-09-15 14:26:29,653:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-15 14:26:29,653:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-15 14:26:29,653:INFO: common: created directory at: artifacts]
[2024-09-15 14:26:29,654:INFO: common: created directory at: artifacts/prepare_base_model]
[2024-09-15 14:26:29,833:WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-15 14:26:29,875:WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-15 14:26:29,895:INFO: main: >>>>>>> stage Prepare Base model completed <<<<<<<

x==========x]
[2024-09-15 14:26:29,895:INFO: main: ********************]
[2024-09-15 14:26:29,895:INFO: main: >>>>>>> stage Training started <<<<<<<]
[2024-09-15 14:26:29,896:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-15 14:26:29,896:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-15 14:26:29,896:INFO: common: created directory at: artifacts]
[2024-09-15 14:26:29,896:INFO: common: created directory at: artifacts/training]
[2024-09-15 14:26:29,950:WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-09-15 14:29:14,844:WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-15 14:29:14,869:INFO: main: >>>>>>> stage Training completed <<<<<<<

x==========x]
[2024-09-15 14:29:14,869:INFO: main: ********************]
[2024-09-15 14:29:14,869:INFO: main: >>>>>>> stage Evaluation stage started <<<<<<<]
[2024-09-15 14:29:14,871:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-15 14:29:14,872:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-15 14:29:14,872:INFO: common: created directory at: artifacts]
[2024-09-15 14:29:15,051:WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-09-15 14:29:24,816:INFO: common: json file saved at: scores.json]
[2024-09-15 14:29:24,817:INFO: common: json file saved at: scores.json]
[2024-09-15 14:29:46,054:INFO: main: >>>>>>> stage Evaluation stage completed <<<<<<<

x==========x]
[2024-09-15 14:34:30,365:INFO: main: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2024-09-15 14:34:30,365:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-15 14:34:30,366:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-15 14:34:30,366:INFO: common: created directory at: artifacts]
[2024-09-15 14:34:30,366:INFO: common: created directory at: artifacts/data_ingestion]
[2024-09-15 14:34:30,366:INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1YKuwii_lywWI_03kOtWC8HF7xy1w2-93/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-09-15 14:34:46,429:INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1YKuwii_lywWI_03kOtWC8HF7xy1w2-93/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-09-15 14:34:46,896:INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<<

x==========x]
[2024-09-15 14:34:46,896:INFO: main: ********************]
[2024-09-15 14:34:46,896:INFO: main: >>>>>>> stage Prepare Base model started <<<<<<<]
[2024-09-15 14:34:46,897:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-15 14:34:46,898:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-15 14:34:46,898:INFO: common: created directory at: artifacts]
[2024-09-15 14:34:46,898:INFO: common: created directory at: artifacts/prepare_base_model]
[2024-09-15 14:34:47,074:WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-15 14:34:47,125:WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-15 14:34:47,146:INFO: main: >>>>>>> stage Prepare Base model completed <<<<<<<

x==========x]
[2024-09-15 14:34:47,146:INFO: main: ********************]
[2024-09-15 14:34:47,147:INFO: main: >>>>>>> stage Training started <<<<<<<]
[2024-09-15 14:34:47,147:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-15 14:34:47,148:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-15 14:34:47,148:INFO: common: created directory at: artifacts]
[2024-09-15 14:34:47,148:INFO: common: created directory at: artifacts/training]
[2024-09-15 14:34:47,192:WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-09-15 14:56:45,256:WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-15 14:56:45,282:INFO: main: >>>>>>> stage Training completed <<<<<<<

x==========x]
[2024-09-15 14:56:45,282:INFO: main: ********************]
[2024-09-15 14:56:45,282:INFO: main: >>>>>>> stage Evaluation stage started <<<<<<<]
[2024-09-15 14:56:45,283:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-15 14:56:45,284:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-15 14:56:45,284:INFO: common: created directory at: artifacts]
[2024-09-15 14:56:45,407:WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-09-15 14:56:55,120:INFO: common: json file saved at: scores.json]
[2024-09-15 14:56:55,121:INFO: common: json file saved at: scores.json]
[2024-09-15 14:57:12,743:INFO: main: >>>>>>> stage Evaluation stage completed <<<<<<<

x==========x]
[2024-09-15 17:23:27,609:INFO: main: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2024-09-15 17:23:27,610:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-15 17:23:27,611:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-15 17:23:27,611:INFO: common: created directory at: artifacts]
[2024-09-15 17:23:27,611:INFO: common: created directory at: artifacts/data_ingestion]
[2024-09-15 17:23:27,611:INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1YKuwii_lywWI_03kOtWC8HF7xy1w2-93/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-09-15 17:23:49,181:INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1YKuwii_lywWI_03kOtWC8HF7xy1w2-93/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-09-15 17:23:49,547:INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<<

x==========x]
[2024-09-15 17:23:49,547:INFO: main: ********************]
[2024-09-15 17:23:49,547:INFO: main: >>>>>>> stage Prepare Base model started <<<<<<<]
[2024-09-15 17:23:49,548:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-15 17:23:49,548:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-15 17:23:49,548:INFO: common: created directory at: artifacts]
[2024-09-15 17:23:49,549:INFO: common: created directory at: artifacts/prepare_base_model]
[2024-09-15 17:23:49,701:WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-15 17:23:49,730:WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-15 17:23:49,748:INFO: main: >>>>>>> stage Prepare Base model completed <<<<<<<

x==========x]
[2024-09-15 17:23:49,748:INFO: main: ********************]
[2024-09-15 17:23:49,748:INFO: main: >>>>>>> stage Training started <<<<<<<]
[2024-09-15 17:23:49,748:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-15 17:23:49,749:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-15 17:23:49,749:INFO: common: created directory at: artifacts]
[2024-09-15 17:23:49,749:INFO: common: created directory at: artifacts/training]
[2024-09-15 17:23:49,789:WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-09-15 17:26:42,512:WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-15 17:26:42,539:INFO: main: >>>>>>> stage Training completed <<<<<<<

x==========x]
[2024-09-15 17:26:42,539:INFO: main: ********************]
[2024-09-15 17:26:42,539:INFO: main: >>>>>>> stage Evaluation stage started <<<<<<<]
[2024-09-15 17:26:42,541:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-15 17:26:42,541:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-15 17:26:42,542:INFO: common: created directory at: artifacts]
[2024-09-15 17:26:42,690:WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-09-15 17:26:53,823:INFO: common: json file saved at: scores.json]
[2024-09-15 17:26:53,824:INFO: common: json file saved at: scores.json]
[2024-09-15 17:27:27,317:INFO: main: >>>>>>> stage Evaluation stage completed <<<<<<<

x==========x]
[2024-09-15 17:37:40,753:INFO: main: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2024-09-15 17:37:40,754:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-15 17:37:40,754:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-15 17:37:40,755:INFO: common: created directory at: artifacts]
[2024-09-15 17:37:40,755:INFO: common: created directory at: artifacts/data_ingestion]
[2024-09-15 17:37:40,755:INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1YKuwii_lywWI_03kOtWC8HF7xy1w2-93/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-09-15 17:38:03,912:INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1YKuwii_lywWI_03kOtWC8HF7xy1w2-93/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-09-15 17:38:04,298:INFO: main: >>>>>>> stage Data Ingestion stage completed <<<<<<<

x==========x]
[2024-09-15 17:38:04,298:INFO: main: ********************]
[2024-09-15 17:38:04,298:INFO: main: >>>>>>> stage Prepare Base model started <<<<<<<]
[2024-09-15 17:38:04,299:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-15 17:38:04,299:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-15 17:38:04,299:INFO: common: created directory at: artifacts]
[2024-09-15 17:38:04,299:INFO: common: created directory at: artifacts/prepare_base_model]
[2024-09-15 17:38:04,462:WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-15 17:38:04,520:WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-15 17:38:04,540:INFO: main: >>>>>>> stage Prepare Base model completed <<<<<<<

x==========x]
[2024-09-15 17:38:04,540:INFO: main: ********************]
[2024-09-15 17:38:04,540:INFO: main: >>>>>>> stage Training started <<<<<<<]
[2024-09-15 17:38:04,540:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-15 17:38:04,541:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-15 17:38:04,541:INFO: common: created directory at: artifacts]
[2024-09-15 17:38:04,541:INFO: common: created directory at: artifacts/training]
[2024-09-15 17:38:04,587:WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-09-15 17:43:46,217:WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-15 17:43:46,324:INFO: main: >>>>>>> stage Training completed <<<<<<<

x==========x]
[2024-09-15 17:43:46,324:INFO: main: ********************]
[2024-09-15 17:43:46,324:INFO: main: >>>>>>> stage Evaluation stage started <<<<<<<]
[2024-09-15 17:43:46,326:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-15 17:43:46,326:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-15 17:43:46,326:INFO: common: created directory at: artifacts]
[2024-09-15 17:43:46,377:WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-09-15 17:43:56,521:INFO: common: json file saved at: scores.json]
[2024-09-15 17:43:56,521:INFO: common: json file saved at: scores.json]
[2024-09-15 17:44:41,255:INFO: main: >>>>>>> stage Evaluation stage completed <<<<<<<

x==========x]
[2024-09-15 19:14:13,986:INFO: stage_01_data_ingestion: >>>>>>> stage Data Ingestion stage started <<<<<<<]
[2024-09-15 19:14:13,987:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-15 19:14:13,989:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-15 19:14:13,990:INFO: common: created directory at: artifacts]
[2024-09-15 19:14:13,990:INFO: common: created directory at: artifacts/data_ingestion]
[2024-09-15 19:14:13,990:INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1YKuwii_lywWI_03kOtWC8HF7xy1w2-93/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-09-15 19:14:31,466:INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1YKuwii_lywWI_03kOtWC8HF7xy1w2-93/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-09-15 19:14:31,832:INFO: stage_01_data_ingestion: >>>>>>> stage Data Ingestion stage completed <<<<<<<

x==========x]
[2024-09-15 19:16:24,071:INFO: stage_02_prepare_base_model: ******************************]
[2024-09-15 19:16:24,071:INFO: stage_02_prepare_base_model: >>>>>>>>>> Stage Prepare Base model started <<<<<<<<<<]
[2024-09-15 19:16:24,072:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-15 19:16:24,073:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-15 19:16:24,074:INFO: common: created directory at: artifacts]
[2024-09-15 19:16:24,074:INFO: common: created directory at: artifacts/prepare_base_model]
[2024-09-15 19:16:24,255:WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-15 19:16:24,292:WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-15 19:16:24,316:INFO: stage_02_prepare_base_model: >>>>>>>>>> Stage Prepare Base model completed <<<<<<<<<<

x==============================x]
[2024-09-15 19:16:26,194:INFO: stage_03_model_training: **********]
[2024-09-15 19:16:26,194:INFO: stage_03_model_training: >>>>>>>>>> stage Training started <<<<<<<<<<]
[2024-09-15 19:16:26,195:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-15 19:16:26,195:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-15 19:16:26,195:INFO: common: created directory at: artifacts]
[2024-09-15 19:16:26,196:INFO: common: created directory at: artifacts/training]
[2024-09-15 19:16:26,261:WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-09-15 19:18:17,926:INFO: stage_03_model_training: **********]
[2024-09-15 19:18:17,926:INFO: stage_03_model_training: >>>>>>>>>> stage Training started <<<<<<<<<<]
[2024-09-15 19:18:17,927:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-15 19:18:17,928:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-15 19:18:17,928:INFO: common: created directory at: artifacts]
[2024-09-15 19:18:17,928:INFO: common: created directory at: artifacts/training]
[2024-09-15 19:18:18,025:WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-09-15 19:23:59,812:WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-15 19:23:59,846:INFO: stage_03_model_training: >>>>> stage Training completed <<<<<

x=====x]
[2024-09-15 19:34:19,571:INFO: stage_04_model_evaluation_mlflow: ***************]
[2024-09-15 19:34:19,571:INFO: stage_04_model_evaluation_mlflow: >>>>>>> stage Evaluation stage started <<<<<<<]
[2024-09-15 19:34:19,572:INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-09-15 19:34:19,573:INFO: common: yaml file: params.yaml loaded successfully]
[2024-09-15 19:34:19,573:INFO: common: created directory at: artifacts]
[2024-09-15 19:34:19,675:WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-09-15 19:34:29,946:INFO: common: json file saved at: scores.json]
[2024-09-15 19:34:29,946:INFO: common: json file saved at: scores.json]
[2024-09-15 19:34:29,946:INFO: stage_04_model_evaluation_mlflow: >>>>>>> stage Evaluation stage completed <<<<<<<

x==========x]
[2024-09-15 20:29:27,259:INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.0.104:8080]
[2024-09-15 20:29:27,259:INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-09-15 20:29:53,285:INFO: _internal: 127.0.0.1 - - [15/Sep/2024 20:29:53] "GET / HTTP/1.1" 200 -]
[2024-09-15 20:30:32,374:ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask/app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask/app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask_cors/extension.py", line 194, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask_cors/decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
                         ^^^^^^^^^^^^^^^^^^
  File "/Users/manasvialam/Documents/ML_projects/Kidney-Disease-Classification/app.py", line 35, in predictRoute
    decodeImage(image,clApp.filename)
  File "/Users/manasvialam/Documents/ML_projects/Kidney-Disease-Classification/src/cnnClassifier/utils/common.py", line 121, in decodeImage
    imgdata = base64.b64ecode(imgstring)
              ^^^^^^^^^^^^^^^
AttributeError: module 'base64' has no attribute 'b64ecode'. Did you mean: 'b64decode'?
[2024-09-15 20:30:32,386:INFO: _internal: 127.0.0.1 - - [15/Sep/2024 20:30:32] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2024-09-15 20:31:12,301:INFO: _internal: 127.0.0.1 - - [15/Sep/2024 20:31:12] "GET / HTTP/1.1" 200 -]
[2024-09-15 20:31:17,423:ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask/app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask/app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask_cors/extension.py", line 194, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask_cors/decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
                         ^^^^^^^^^^^^^^^^^^
  File "/Users/manasvialam/Documents/ML_projects/Kidney-Disease-Classification/app.py", line 35, in predictRoute
    decodeImage(image,clApp.filename)
  File "/Users/manasvialam/Documents/ML_projects/Kidney-Disease-Classification/src/cnnClassifier/utils/common.py", line 121, in decodeImage
    imgdata = base64.b64ecode(imgstring)
              ^^^^^^^^^^^^^^^
AttributeError: module 'base64' has no attribute 'b64ecode'. Did you mean: 'b64decode'?
[2024-09-15 20:31:17,427:INFO: _internal: 127.0.0.1 - - [15/Sep/2024 20:31:17] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2024-09-15 20:32:34,798:INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.0.104:8080]
[2024-09-15 20:32:34,798:INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-09-15 20:32:48,071:INFO: _internal: 127.0.0.1 - - [15/Sep/2024 20:32:48] "GET / HTTP/1.1" 200 -]
[2024-09-15 20:32:52,369:ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask/app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask/app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask_cors/extension.py", line 194, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask_cors/decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
                         ^^^^^^^^^^^^^^^^^^
  File "/Users/manasvialam/Documents/ML_projects/Kidney-Disease-Classification/app.py", line 43, in predictRoute
    decodeImage(image, clApp.filename)
  File "/Users/manasvialam/Documents/ML_projects/Kidney-Disease-Classification/src/cnnClassifier/utils/common.py", line 121, in decodeImage
    imgdata = base64.b64ecode(imgstring)
              ^^^^^^^^^^^^^^^
AttributeError: module 'base64' has no attribute 'b64ecode'. Did you mean: 'b64decode'?
[2024-09-15 20:32:52,380:INFO: _internal: 127.0.0.1 - - [15/Sep/2024 20:32:52] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2024-09-15 20:35:21,623:INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.0.104:8080]
[2024-09-15 20:35:21,623:INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-09-15 20:35:28,487:INFO: _internal: 127.0.0.1 - - [15/Sep/2024 20:35:28] "GET / HTTP/1.1" 200 -]
[2024-09-15 20:35:34,474:ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask/app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask/app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask_cors/extension.py", line 194, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask_cors/decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
                         ^^^^^^^^^^^^^^^^^^
  File "/Users/manasvialam/Documents/ML_projects/Kidney-Disease-Classification/app.py", line 43, in predictRoute
    decodeImage(image, clApp.filename)
  File "/Users/manasvialam/Documents/ML_projects/Kidney-Disease-Classification/src/cnnClassifier/utils/common.py", line 121, in decodeImage
    imgdata = base64.b64encode(imgstring)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/base64.py", line 58, in b64encode
    encoded = binascii.b2a_base64(s, newline=False)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: a bytes-like object is required, not 'str'
[2024-09-15 20:35:34,482:INFO: _internal: 127.0.0.1 - - [15/Sep/2024 20:35:34] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2024-09-16 13:41:32,958:INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.0.104:8080]
[2024-09-16 13:41:32,958:INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-09-16 13:41:46,812:INFO: _internal: 127.0.0.1 - - [16/Sep/2024 13:41:46] "GET / HTTP/1.1" 200 -]
[2024-09-16 13:41:51,497:ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask/app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask/app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask_cors/extension.py", line 194, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask_cors/decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
                         ^^^^^^^^^^^^^^^^^^
  File "/Users/manasvialam/Documents/ML_projects/Kidney-Disease-Classification/app.py", line 43, in predictRoute
    decodeImage(image, clApp.filename)
  File "/Users/manasvialam/Documents/ML_projects/Kidney-Disease-Classification/src/cnnClassifier/utils/common.py", line 121, in decodeImage
    imgdata = base64.b64encode(imgstring)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/base64.py", line 58, in b64encode
    encoded = binascii.b2a_base64(s, newline=False)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: a bytes-like object is required, not 'str'
[2024-09-16 13:41:51,509:INFO: _internal: 127.0.0.1 - - [16/Sep/2024 13:41:51] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2024-09-16 13:42:00,008:INFO: _internal: 127.0.0.1 - - [16/Sep/2024 13:42:00] "[31m[1mGET /predict HTTP/1.1[0m" 405 -]
[2024-09-16 13:42:00,133:INFO: _internal: 127.0.0.1 - - [16/Sep/2024 13:42:00] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-09-16 13:42:14,409:ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask/app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask/app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask_cors/extension.py", line 194, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/site-packages/flask_cors/decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
                         ^^^^^^^^^^^^^^^^^^
  File "/Users/manasvialam/Documents/ML_projects/Kidney-Disease-Classification/app.py", line 43, in predictRoute
    decodeImage(image, clApp.filename)
  File "/Users/manasvialam/Documents/ML_projects/Kidney-Disease-Classification/src/cnnClassifier/utils/common.py", line 121, in decodeImage
    imgdata = base64.b64encode(imgstring)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/kidney/lib/python3.12/base64.py", line 58, in b64encode
    encoded = binascii.b2a_base64(s, newline=False)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: a bytes-like object is required, not 'str'
[2024-09-16 13:42:14,415:INFO: _internal: 127.0.0.1 - - [16/Sep/2024 13:42:14] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2024-09-16 13:44:00,880:INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.0.104:8080]
[2024-09-16 13:44:00,880:INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-09-16 13:44:07,850:INFO: _internal: 127.0.0.1 - - [16/Sep/2024 13:44:07] "GET / HTTP/1.1" 200 -]
[2024-09-16 13:44:11,888:WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-09-16 13:44:12,111:INFO: _internal: 127.0.0.1 - - [16/Sep/2024 13:44:12] "POST /predict HTTP/1.1" 200 -]
